docs/examples/Rocket_Car.rst

```diff
@@ -0,0 +1,1585 @@
+##
+
+.. raw:: html
+
+   <center>
+
+:math:`\mathbf{\text{Modeling of a Variable-Mass Nonholonomic Gyrostatic Rocket Car Using Extended Kane's Equations}}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Ge, Z.M. and Cheng, Y.H., 1982. Extended Kane’s equations for nonholonomic variable mass system.}`
```

Comment:
Make a rst citation for this:

https://docutils.sourceforge.io/docs/user/rst/quickref.html#citations